_posts/2020-04-15-cockpit-217.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 217
+author: mvo
+date: 2020-04-15
+tags: cockpit linux
+slug: cockpit-217
+category: release
+summary: Cockpit with improved ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 217.
+
+### More details about Insights rule hits
+
+If your machine is connected to Red Hat Insights, the Health card on
+the Overview page will now show more details.
+
+![Insights details](/images/health-insights.png)
+
+### Buttons are now left aligned
+
+In accordance with Patternfly (XXX-LINK-XXX), the buttons in all modal
+dialogs have been left aligned.
```

Comment:
And more importantly, Cancel and other secondary buttons have been de-emphasized to be link buttons. (That's not release note ready text, of course).

Reply:
Fixed.